js/stringProcessing/getSentences.js

```diff
@@ -36,7 +39,7 @@ var invalidateOnCapital = function( text, positions, i ) {
 	var firstChar = text.substring( positions[ i ] + 1, positions[ i ] + 2 );
 
 	// If a sentence starts with a number or a whitespace, it shouldn't invalidate
-	if ( firstChar === firstChar.toLocaleLowerCase() && isNaN( parseInt( firstChar, 10 ) ) && firstChar.match( /[\s<]/ ) === null ) {
+	if ( firstChar === firstChar.toLocaleLowerCase() && isNaN( parseInt( firstChar, 10 ) ) && firstChar.match( sentenceEndingRegex ) === null ) {
```

Comment:
It might be nicer to replace this with a single method call or multiple methods that each contain their own little bit of logic.

This would make the code a lot more readable.